versions.yaml

```diff
@@ -153,16 +153,15 @@ assets:
   kernel:
     description: "Linux kernel optimised for virtual machines"
     url: "https://cdn.kernel.org/pub/linux/kernel/v5.x/"
-    version: "v5.15.48"
+    version: "v5.19"
```

Comment:
I'd prefer one commit bumping the kernel version, and another commit bumping the version for SEV, it'd be cleaner which change is related to each one of the targets.

Reply:
done

Reply:
Also, please, update to v5.19.1

Reply:
done